### PR TITLE
BUG FIX - new checkbox fixes

### PR DESCRIPTION
**Pull request by Nathan-O**
fixed issues blocking checkbox from working properly

### CODE REVIEW

**Comment by Samsinite** [comment]
Feel free to also bump the package.json version to `v1.0.0-alpha.2`, and go a head and create that tag after this is merged.

File: addon/templates/components/bootstrap-checkbox.hbs [hunk 1]
```diff
@@ -3,7 +3,7 @@
     {{yield
       (hash
         label=(component control.label for=control.inputId srOnly=srOnly label=label)
-        input=(component 'bootstrap/-input'
+        input=(component 'bootstrap/-checkbox'
```
Comment (harokevin) [inline addon/templates/components/bootstrap-checkbox.hbs]: On line 24 below this comment the following is defined:
```
{{bootstrap/-input
       tabindex=tabindex
       checked=isChecked
       onClick=(action "change")
       elementId=control.inputId
       type='checkbox'
       placeholder=placeholder
       readonly=readonly
       disabled=disabled
       key-press=key-press
       key-up=key-up
       key-down=key-down
       required=required
     }}
```

should this also be 'bootstrap/-checkbox' and not 'bootstrap/-input'?

Comment (Samsinite) [reply addon/templates/components/bootstrap-checkbox.hbs]: Yes it should, great catch Kevin.

File: addon/components/bootstrap/-checkbox.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember';
+
+export default Ember.Checkbox.extend({
+  classNames: ['form-control'],
+
+  click() {
+  	const onClick = this.get('onClick');
```
Comment (Samsinite) [inline addon/components/bootstrap/-checkbox.js]: Guess the previous comment wasn't very clear. The documentation (and prior versions) used `onclick`, we should keep this name so we aren't breaking other stuff, so this should be `this.get('onclick')`.

File: addon/templates/components/bootstrap-checkbox.hbs [hunk 3]
```diff
@@ -3,9 +3,9 @@
...
           checked=isChecked
-          onClick=(action "change")
+          onclick=(action "change")
```
Comment (Samsinite) [inline addon/templates/components/bootstrap-checkbox.hbs]: There is no `change` action in this component. This should be spitting an error into your console.

File: addon/templates/components/bootstrap-checkbox.hbs [hunk 4]
```diff
@@ -21,10 +21,10 @@
     }}
   {{else}}
     {{control.label for=control.inputId label=label srOnly=srOnly}}
-    {{bootstrap/-input
+    {{bootstrap/-checkbox
       tabindex=tabindex
       checked=isChecked
-      onClick=(action "change")
+      onclick=(action "change")
```
Comment (Samsinite) [inline addon/templates/components/bootstrap-checkbox.hbs]: There is no `change` action in this component. This should be spitting an error into your console.

**Pull request merged**